terraform/install.sh

```diff
@@ -174,6 +174,24 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
+  while [ -z $external_ip ]; do
+     external_ip=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); 
+     [ -z "$external_ip" ] && sleep 30; 
+  done;
+
+  gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
+  log "Please create a monitoring workspace for the project by clicking on the following link: $gcp_monitoring_path"
+  read -p "When you are done, please press enter to continue"
+
+  log "Creating monitoring examples (dashboards, uptime checks, alerting policies, etc.)..."
+  cd ./monitoring
+  terraform init -lock=false
+  terraform apply --auto-approve -var="project_id=${project_id}" -var="external_ip=${external_ip}"
+  cd ..
```

Comment:
I don't like seeing relative cd commands like this, it can cause hard to catch bugs. `pushd` and `popd` could work well here

Reply:
Changed to pushd/popd.